src/utils/rates.js

```diff
@@ -107,10 +109,46 @@ export const fiatInvestmentBalance = (balance: BigNumber, rates: RatesPerChain,
   return fiatAmount?.toFixed(2);
 };
 
-export const fiatTokenValue = (tokenValue: number, rates: RatesPerChain, currency: Currency, decimals?: ?number) => {
+export const fiatTokenValue = (tokenValue: number, rates: RatesPerChain, currency: Currency) => {
+  if (isNaN(tokenValue)) return '';
   const nativeAssetRate = rates[nativeAssetPerChain[CHAIN.ETHEREUM].address];
+  const currencySymbol = getCurrencySymbol(currency);
 
   const fiatAmount = (tokenValue * nativeAssetRate?.[currency]) / nativeAssetRate?.[USD];
 
-  return fiatAmount?.toFixed(decimals || 2);
+  const decimalsFiatValue = fiatAmount?.toFixed(fiatAmount > 1 ? 2 : 4);
```

Comment:
Just a question, in what scenario do we need `toFixed` precision to be `4`?

Reply:
if fiat value is less than 1. like 0.0553 
if more than 1. like 1.56

Reply:
Got it - cheers